editor/editor_log.h

```diff
@@ -68,7 +74,7 @@ class EditorLog : public VBoxContainer {
 	void _notification(int p_what);
 
 public:
-	void add_message(const String &p_msg, bool p_error = false);
+	void add_message(const String &p_msg, int p_type = MSG_TYPE_STD);
```

Comment:
I think you could use `MessageType p_type = MSG_TYPE_STD` instead of `int`, but otherwise looks all good :+1:

Reply:
Eh.. If I change it to MessageType its required to change it in more places in code(in order to compile) example
![image](https://user-images.githubusercontent.com/3036176/44042889-29858fcc-9f2a-11e8-8f30-120d1a57eb3a.png)
Do you still want me to change it ?

Reply:
Wait, I will change it..

Reply:
I cannot change it right now, because it requires to change it in other parts of code. Could it be merged as its now ?

Reply:
I could remove MessageType name and make a nameless enum. Is it good solution ?